plugins/aggregators/derivative/README.md

```diff
@@ -0,0 +1,72 @@
+# Derivative Aggregator Plugin
+The Derivative Aggregator Plugin estimates the derivative for all fields of the
+aggregated metrics.
+
+### Time Derivatives
+
+In its default configuration it determines the first and last measurement of
+the period. From these measurements the time difference in seconds is
+calculated. This time difference is than used to divide the difference of each
+field using the following formula:
+```
+              field_last - field_first
+derivative = --------------------------
+                  time_difference
+```
+For each field the derivative is emitted with a naming pattern
+`fieldname_by_seconds`.
+
+### Custom Derivation Variable
+
+The plugin supports to use a field of the aggregated measurements as derivation
+variable in the denominator. This variable is assumed to be a monotonously
+increasing value. In this feature the following formula is used:
+```
+                 field_last - field_first
+derivative = --------------------------------
+              variable_last - variable_first
+```
+For each field the derivative then is emitted with a naming pattern
+`fieldname_by_variablename`.
+
+### Roll-Over to next Period
+
+By default the last measurement is used as first measurement in the next
+aggregation period. This enables a continuous calculation of the derivative. If
+within the next period an earlier timestamp is encountered this measurement will
+replace the roll-over metric. A main benefit of this roll-over is the ability to
+cope with multiple "quiet" periods, where no new measurement is pushed to the
+aggregator. The roll-over will take place at most `max_roll_over`times.
+
+### Configuration
+
+```toml
+[[aggregators.derivative]]
+  ## Specific Derivative Aggregator Arguments:
+  ## Configure a custom derivation variable. Timestamp is used if none is given.
+  # variable = "fieldname_of_derivation_vaiable"
+  ## Infix to separate field name from derivation variable.
+  infix = "_by_"  # default: _by_
+  ## Roll-Over last measurement to first measuremen of next period
+  max_roll_over = 10 # default: 10
+
+  ## General Aggregator Arguments:
```

Comment:
Another general argument is `drop_original`. Please add this here.

Reply:
I don't think this is resolved!?

Reply:
Thanks, please also add in `sampleConfig`. Put this as a separate comment, so you can mark this one as resolved.